apps/appointments/src/pages/admin/addService/styles/index.module.scss

```
@@ -0,0 +1,117 @@
+@use "@carbon/react/scss/spacing" as *;
+@use "@carbon/react/scss/type" as *;
+@use "@carbon/react/scss/theme" as *;
+
+.noPrivilegeContainer {
+  margin-block: $spacing-10;
+  margin-inline: calc($spacing-12 + $spacing-05);
+  box-shadow: 0px 2px 6px 0px $shadow;
+  padding: $spacing-10;
+  @include type-style("body-compact-01");
+  color: $support-error;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)